p2p/libp2p/netMessenger.go

```diff
@@ -365,7 +365,7 @@ func (netMes *networkMessenger) CreateTopic(name string, createChannelForTopic b
 	_, found := netMes.topics[name]
 	if found {
 		netMes.mutTopics.Unlock()
-		return p2p.ErrTopicAlreadyExists
+		return nil
```

Comment:
why masking error ?

Reply:
it's not error masking. After discussions with Iulian, in order that our epoch start component to work, we need that network messenger won't fail if we try to register twice for the same topic. Right now we register a processor for a topic inside epoch start data provider and the unregister it so when all stuff will start we can register again a processor (the real one) for the topic